bigchaindb/util.py

```diff
@@ -501,6 +501,26 @@ def get_hash_data(transaction):
     return crypto.hash_data(serialize(tx))
 
 
+def verify_vote_signature(signed_vote):
```

Comment:
You mean `voters` as in the voter list that comes with the vote? Couldn't that be faked too?